mlflow/server/js/src/utils/Utils.test.js

```diff
@@ -1,83 +1,83 @@
 import Utils from './Utils'
 
 test("formatMetric", () => {
```

Comment:
Changed all the toBe into toEqual because toBe is for object identity and may not always work depending on the JS engine implementation.